nano/node/CMakeLists.txt

```diff
@@ -21,6 +21,8 @@ add_library (node
 	cli.cpp
 	common.cpp
 	common.hpp
+	confirmation_height_processor.hpp
```

Comment:
Probably consistent name for files without _

Reply:
Thanks, changed

Reply:
I don't think confirmationheightprocessor.cpp is readable. We use underscores a few other places already, like in tests and ipc_client.hpp in the lib.

Reply:
@cryptocode probably you right...

Reply:
gladly reverted back :)